package.json

```diff
@@ -65,15 +64,17 @@
     "prettier": "2.8.8",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "storybook": "^7.6.20",
-    "ts-node":"10.9.2",
+    "storybook": "^8.5.3",
+    "ts-node": "10.9.2",
     "turbo": "^2.0.6",
     "typescript": "~4.9.0"
   },
   "pnpm": {
     "overrides": {
       "caniuse-lite": "^1.0.30001585",
-      "jackspeak": "2.1.1"
+      "jackspeak": "2.1.1",
+      "@testing-library/dom": "9.3.4",
+      "@testing-library/user-event": "13.5.0"
```

Comment:
@bruugey `@storybook/test` was resolving an incompatible version of `@testing-library/dom`

Reply:
ah ok